### PR TITLE
(feat) Ability export profile data

### PR DESCRIPTION
**Pull request by PrimalPimmy**
New Feature added for profile! Now we can use the flag `karmor profile --save` to run profile, and as it runs, it sends the data to 4 separate json files for File, Process, Network and Summary respectively. All these files are stored in a common folder named `Profile_Summary`

### CODE REVIEW

File: profile/Client/profileClient.go [hunk 1]
```diff
@@ -425,8 +452,30 @@ func generateRowsFromData(data []pb.Log, Operation string) []table.Row {
...
+		if Operation == "File" {
+			convertToJson("File", jsondata)
+		} else if Operation == "Process" {
+			convertToJson("Process", jsondata)
+		} else if Operation == "Network" {
+			convertToJson("Network", jsondata)
+		} else if Operation == "Syscall" {
+			convertToJson("Syscall", jsondata)
+		}
+	}
```
Comment (daemon1024) [inline profile/Client/profileClient.go:477]: How does this work? Will it only save when we are watching it?

Comment (PrimalPimmy) [reply profile/Client/profileClient.go:477]: yup, if `--save` flag is there, then only it will watch and save it

Comment (daemon1024) [reply profile/Client/profileClient.go:477]: Not that, does it save a single operation at a time?

Comment (PrimalPimmy) [reply profile/Client/profileClient.go:477]: Nope, it saves all of them together.

Comment (PrimalPimmy) [reply profile/Client/profileClient.go:477]: ref for generating table: https://github.com/kubearmor/kubearmor-client/blob/main/profile/Client/profileClient.go#L230

Comment (daemon1024) [reply profile/Client/profileClient.go:477]: Thanks, Gotit.

**Pull request merged**